src/Main.elm

```diff
@@ -45,6 +51,7 @@ init flags url key =
 type Msg
     = UrlChanged Url.Url
     | LinkClicked Browser.UrlRequest
+    | ToggleLanguage
```

Comment:
It's maybe trivial but the convention is to write update names like an action the has taken place, like `LanguageChangeRequested` `LanguageChangerClicked`